indra/newview/llviewerjoystick.cpp

```diff
@@ -195,7 +195,7 @@ BOOL CALLBACK di8_devices_callback(LPCDIDEVICEINSTANCE device_instance_ptr, LPVO
                 LLSD::Binary data; //just an std::vector
                 data.resize(size);
                 memcpy(&data[0], &device_instance_ptr->guidInstance /*POD _GUID*/, size);
-                LLViewerJoystick::getInstance()->initDevice(&device, product_name, LLSD(data));
+                LLViewerJoystick::getInstance()->initDevice((void*)&device, product_name, LLSD(data));
```

Comment:
? I thought every `T*` could implicitly convert to `void*`?

Reply:
Ah yeah missed that slipping in to my staged files.